daml-lf/transaction-test-lib/src/main/scala/lf/value/test/ValueGenerators.scala

```diff
@@ -76,7 +77,7 @@ object ValueGenerators {
 
   val nameGen: Gen[Name] = {
     val firstChars =
-      "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ$_".toVector
+      """abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ_$""".toVector
```

Comment:
Doesn't look like a particularly meaningful change.

Reply:
without inverting '$' and '_' the compiler is complaining: 

``
    ERROR: /home/remy/work/daml/daml-lf/transaction-test-lib/BUILD.bazel:10:1: scala //daml-lf/transaction-test-lib:transaction-test-lib failed (Exit 1)
daml-lf/transaction-test-lib/src/main/scala/lf/value/test/ValueGenerators.scala:80: warning: possible missing interpolator: detected interpolated identifier `$NodeId`
      "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ$_".toVector
``  

because of the change I made in the import:
``
import com.daml.lf.value.Value.{NodeId => _, _} 
``

I assume this is a bug.